packages/react-instantsearch-dom/src/components/InfiniteHits.js

```diff
@@ -62,7 +62,11 @@ InfiniteHits.propTypes = {
   refineNext: PropTypes.func.isRequired,
   translate: PropTypes.func.isRequired,
   className: PropTypes.string,
-  hitComponent: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+  hitComponent: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.func,
+    PropTypes.object
```

Comment:
```suggestion
    PropTypes.object // support components wrapped with React.memo
```

Reply:
is this comment needed? Alternatively I also thought naming it something like ReactElement so it can be reused